beebop/app.py

```diff
@@ -481,13 +499,15 @@ def get_network_graphs(p_hash) -> json:
         cluster_result = get_cluster_assignments(p_hash, storage_location)
         graphmls = {}
         for cluster_info in cluster_result.values():
-            raw_cluster_num = cluster_info["raw_cluster_num"]
+            cluster = cluster_info["cluster"]
```

Comment:
Why has this changed?

Reply:
as discussed as output folder naming has changed now